gensim/models/experimental/custom_callbacks.py

```diff
@@ -0,0 +1,72 @@
+import logging
```

Comment:
General question: is this have the sense to have distinct files for callbacks/layers/losses here? Why not place it directly in model file? Or, at least, join all of it to one file like utils?